pkg/master/master.go

```diff
@@ -85,6 +90,20 @@ const (
 	// DefaultEndpointReconcilerInterval is the default amount of time for how often the endpoints for
 	// the kubernetes Service are reconciled.
 	DefaultEndpointReconcilerInterval = 10 * time.Second
+	// DefaultEndpointReconcilerTTL is the default TTL timeout for the storage layer
+	DefaultEndpointReconcilerTTL = 15
+)
+
+// EndpointReconcilerEnum selects which reconciler to use
+type EndpointReconcilerEnum int
+
+const (
+	// DefaultMasterCountReconciler will select the original reconciler
+	MasterCountReconciler = "master-count"
```

Comment:
Oh, you already have these constants! give them a type and use them everywhere. :)

Reply:
(you may need to put them in a more visible place)

Reply:
Probably should make them CamelCase to be consistent with other constants for type

Reply:
The problem was the circular dependency it caused, but I'll look into it.